### PR TITLE
style: add link-list.row-gap token and other link list tweaks

### PR DESCRIPTION
**Pull request by Robbert**
This is part of getting the [Link list in the NL Design System community Figma](https://www.figma.com/design/gX3fq7k0uUKC45AJbuItDv/Local---Voorbeeld---Bibliotheek?node-id=1548-6863&t=KuOvWHVyjBq0moFp-4) consistent with the code from the Utrecht community component.

See also: [Link list GitHub discussion at NL Design System](https://github.com/orgs/nl-design-system/discussions/266)

### CODE REVIEW

File: components/link-list/src/_mixin.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  --utrecht-link-icon-size: var(--utrecht-link-list-icon-size);
+  --utrecht-icon-inset-block-start: var(--utrecht-link-list-icon-inset-block-start);
+
+  display: block;
+  font-weight: var(--utrecht-link-list-item-font-weight);
+}
+
+@mixin utrecht-link-list__link {
+  align-items: baseline;
+  column-gap: var(--utrecht-link-list-item-column-gap);
```
Comment (Yolijn) [inline components/link-list/src/_mixin.scss:42]: Should this be `--utrecht-link-list-link-column-gap` or should `column-gap` be set in the mixin for the `__item` instead?

Comment (Yolijn) [reply components/link-list/src/_mixin.scss:42]: Also I'm wondering what the difference is between een link-list__item and a link-list__link... the React component also only implements the `LinkListLink`?

Comment (Robbert) [reply components/link-list/src/_mixin.scss:42]: you make some very good points

Comment (Robbert) [reply components/link-list/src/_mixin.scss]: done

**Review by Yolijn — APPROVED** [review]

**Pull request merged**